scripts/eval/yamls/coding_tasks.yaml

```diff
@@ -4,63 +4,62 @@ icl_tasks:
   dataset_uri: eval/local_data/programming/human_eval.jsonl # ADD YOUR OWN DATASET URI
   num_fewshot: [0]
   pass_at_k: 1
-  num_beams: 20
+  num_beams: 5
```

Comment:
Is this reasonable?

Reply:
Yes, this is basically the largest value I could get and still have 30B models run on 16 40GB A100s. It raises the variance of the measurements, but it's faster and less memory intensive